### ISSUE TITLE
Failed NPM Install - exports problem?

### ISSUE THREAD

**Issue by cycle4passion** [leveluptuts/svelte-toy#5]
I understand it's alpha, and in flux, just a FYI

**Environment**: Svelte+TS

version 0.1.14 works in [REPL](https://svelte.dev/repl/70f6498e5e8046aaa610231279dca314?version=3.46.4)
fails in VSCode
fails in [Stackblitz](https://stackblitz.com/edit/vitejs-vite-cnsxhq?file=src%2FApp.svelte&terminal=dev)

**Stackblitz steps**:

Start new Svelte + TS project
npm i @leveluptuts/svelte-toy
paste in demo to App.svelte

```js
<script>
	import Toy from '@leveluptuts/svelte-toy';
	import { writable } from 'svelte/store';

	let example = writable({
		name: 'John',
		isActive: false
	});
</script>

<h1>Hi, my name is {$example.name}</h1>

{#if $example.isActive}
	<p>I'm active</p>
{/if}

<Toy register={[{ example }]} />
```

**Error**: [plugin:vite:import-analysis] Failed to resolve entry for package "@leveluptuts/svelte-toy". The package may have incorrect main/module/exports specified in its package.json: Failed to resolve entry for package "@leveluptuts/svelte-toy". The package may have incorrect main/module/exports specified in its package.json.

**Comment by maku** [comment]
Yeah I have the same problem. Here is my simple repl https://svelte.dev/repl/4c36a8e491c24d11b44a4b09a65a3795?version=3.48.0 @stolinski

**Comment by stolinski** [comment]
Just pushed v2. `import { Toy } from '@leveluptuts/svelte-toy'` is now the correct import

**Comment by cycle4passion** [comment]
Now errors with 'store' is not a store with a 'subscribe' method" in Svelte REPL. In Stackblitz the error is "Failed to resolve entry for package "@leveluptuts/svelte-toy". The package may have incorrect main/module/exports specified in its package.json: Failed to resolve entry for package "@leveluptuts/svelte-toy". The package may have incorrect main/module/exports specified in its package.json."

**Comment by stolinski** [comment]
The repl may have changed. I updated the readme to have the new error https://svelte.dev/repl/4566ac90a5ca42d6ade1c383b6f1ba66?version=3.43.1

The api has changed slightly so the store error is because of that that change. The main prop is now ={{storeName, storeName2}} rather than ={[{storeName}, {storeName2}]}. Double check you are using v2.0.0
